package.json

```diff
@@ -31,8 +31,8 @@
         "jasmine": "^2.6.0",
         "jest": "20.0.4",
         "lint-staged": "~4.0.1",
-        "mobx": "~2.6.5",
-        "mobx-react": "~4.0.3",
+        "mobx": "^4.3.1",
```

Comment:
I'll bump it in a separate change.  (My plan is to do a beta release initially.)

We'll definitely have to maintain a v3 branch since there are some subtle but significant changes in MobX v4 and it's not going to be feasible to immediately upgrade.  But v3 has been pretty stable, so I don't expect to need to maintain much.